### PR TITLE
Add Consumer Metricsfilter and some consumerMetricsKey

### PR DESCRIPTION
**Pull request by lcb11**
## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by songxiaosheng** [comment]
please use English

**Comment by lcb11** [comment]
> please use English

ok

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.apache.dubbo.rpc.model.ApplicationModel;
+import org.apache.dubbo.rpc.model.ScopeModelAware;
+
+import java.util.function.Consumer;
+
+import static org.apache.dubbo.common.constants.CommonConstants.CONSUMER;
+
+
+@Activate(group =CONSUMER, order = -1)
+public class MetricsFilterConsumer implements Filter, BaseFilter.Listener, ScopeModelAware {
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java:16]: Replace with `ClusterFilter`

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package org.apache.dubbo.metrics.filter;
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java:1]: Please add ASF license header

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+import org.apache.dubbo.common.extension.Activate;
+
+import org.apache.dubbo.metrics.collector.DefaultMetricsCollector;
+import org.apache.dubbo.rpc.*;
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java:6]: do not use comma import

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+
+@Activate(group =CONSUMER, order = -1)
+public class MetricsFilterConsumer implements Filter, BaseFilter.Listener, ScopeModelAware {
+
+    DefaultMetricsCollector collector=null;
+    ApplicationModel applicationModel=null;
+
+    @Override
+    public void setApplicationModel(ApplicationModel applicationModel) {
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java:22]:
```suggestion
    public MetricsFilterConsumer(ApplicationModel applicationModel) {
```

File: dubbo-test/dubbo-test-check/pom.xml [hunk 5]
```diff
@@ -83,6 +83,12 @@
             <artifactId>async-http-client</artifactId>
             <version>${async.http.client.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-common</artifactId>
+            <version>3.2.0-beta.4</version>
+            <scope>compile</scope>
+        </dependency>
```
Comment (AlbumenJ) [inline dubbo-test/dubbo-test-check/pom.xml:91]: What is the purpose of changing this

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java (hunk 1 above)
Comment (lcb11) [reply dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java:16]: ok

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java (hunk 2 above)
Comment (lcb11) [reply dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java:1]: ok

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java (hunk 3 above)
Comment (lcb11) [reply dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MetricsFilterConsumer.java:6]: ok, I will change this

File: dubbo-test/dubbo-test-check/pom.xml (hunk 5 above)
Comment (lcb11) [reply dubbo-test/dubbo-test-check/pom.xml:91]: sorry,  I didn't know this was change.

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/ClusterFilter.java [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+import java.util.function.Consumer;
+
+import static org.apache.dubbo.common.constants.CommonConstants.CONSUMER;
+
+
+@Activate(group = CONSUMER, order = -1)
+public class ClusterFilter implements Filter, BaseFilter.Listener, ScopeModelAware {
+
+    DefaultMetricsCollector collector=null;
+    ApplicationModel applicationModel=null;
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/ClusterFilter.java:40]:
```suggestion
    private final DefaultMetricsCollector collector;
    private final ApplicationModel applicationModel;
```

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/ClusterFilter.java (hunk 6 above)
Comment (lcb11) [reply dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/ClusterFilter.java:40]: ok

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/ClusterFilter.java [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+import org.apache.dubbo.rpc.Result;
+import org.apache.dubbo.rpc.RpcException;
+import org.apache.dubbo.rpc.model.ApplicationModel;
+import org.apache.dubbo.rpc.model.ScopeModelAware;
+
+import static org.apache.dubbo.common.constants.CommonConstants.CONSUMER;
+
+
+@Activate(group = CONSUMER, order = -1)
+public class ClusterFilter implements Filter, BaseFilter.Listener, ScopeModelAware {
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/ClusterFilter.java:35]: Rename.
`ClusterFilter` is duplicated.

File: dubbo-metrics/dubbo-metrics-default/src/test/java/org/apache/dubbo/metrics/filter/MetricsFilterTest.java [hunk 8]
```diff
@@ -26,11 +26,7 @@
 import org.apache.dubbo.metrics.model.MetricsKey;
 import org.apache.dubbo.metrics.model.sample.GaugeMetricSample;
 import org.apache.dubbo.metrics.model.sample.MetricSample;
-import org.apache.dubbo.rpc.AppResponse;
-import org.apache.dubbo.rpc.Invoker;
-import org.apache.dubbo.rpc.Result;
-import org.apache.dubbo.rpc.RpcException;
-import org.apache.dubbo.rpc.RpcInvocation;
+import org.apache.dubbo.rpc.*;
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-default/src/test/java/org/apache/dubbo/metrics/filter/MetricsFilterTest.java:29]: Do not use comma import.
You can import code style config in `codestyle` folder.

File: dubbo-rpc/dubbo-rpc-triple/pom.xml [hunk 9]
```diff
@@ -92,6 +92,11 @@
             <version>${reactor.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo</artifactId>
+            <version>3.2.0-beta.4</version>
+        </dependency>
```
Comment (AlbumenJ) [inline dubbo-rpc/dubbo-rpc-triple/pom.xml:99]: `dubbo-rpc-triple` should not depends on `dubbo-all`

File: dubbo-test/dubbo-test-check/pom.xml (hunk 5 above)
Comment (AlbumenJ) [inline dubbo-test/dubbo-test-check/pom.xml:91]: Still exist

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

**Review by songxiaosheng — CHANGES_REQUESTED** [review]

**Pull request closed without merging**